exporter/opentelemetry-exporter-jaeger/tests/test_jaeger_exporter_thrift.py

```diff
@@ -81,32 +98,37 @@ def test_constructor_explicit(self):
         username = "username"
         password = "password"
         auth = (username, password)
-
-        exporter = jaeger_exporter.JaegerExporter(
-            service_name=service,
-            agent_host_name=agent_host_name,
-            agent_port=agent_port,
-            collector_endpoint=collector_endpoint,
-            username=username,
-            password=password,
-            max_tag_value_length=42,
+        env_patch = patch.dict(
```

Comment:
I'm instantiating a TraceProvider with an actual Resource with explicit service_name in test_export() function.

Since currently we can instantiate only one instance of TracerProvider (sdk.trace), i just did it only in test_export function.
method set_trace_provider() currently throws a warning and returns if code tries to instantiate multiple tracerprovider()
```
    if _TRACER_PROVIDER is not None:
        logger.warning("Overriding of current TracerProvider is not allowed")
        return
```

Reply:
Tests should not keep any state and should be independent of each other. I would create/destroy the tracer_provider between each test case.

Reply:
i have made the changes by patching the _TRACER_PROVIDER = None before instantiating TracerProvider()